data/boarding missions.txt

```diff
@@ -366,6 +354,7 @@ mission "Assisting Free Worlds (Join)"
 		random < 15
 	source
 		government "Free Worlds"
+		has "event: joined the free worlds"
```

Comment:
I think this should be `not "event: joined the free worlds"`

Reply:
fair point